src/util/timestamper.cpp

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************\
+
+Module: Timestamps
+
+Author: Kareem Khazem <karkhaz@karkhaz.com>
+
+\*******************************************************************/
+
+
+#include <chrono>
+#include <cstdlib>
+
+#include "invariant.h"
+#include "timestamper.h"
```

Comment:
Nit pick: I think we've move to including the header file associated with an implementation to the top of the list of includes.